projects/components/src/table/cells/types/core-table-cell-parser-type.ts

```diff
@@ -4,5 +4,6 @@ export const enum CoreTableCellParserType {
   Number = 'number',
   String = 'string',
   Timestamp = 'timestamp',
-  Icon = 'icon'
+  Icon = 'icon',
+  Object = 'object'
```

Comment:
This value would update too. Do you prefer the name generic over NoOp ? I feel like Noop describes what it's doing better